saleor/graphql/order/types.py

```diff
@@ -1230,18 +1269,26 @@ class Order(ModelObjectType[models.Order]):
         required=True,
         deprecation_reason=(f"{DEPRECATED_IN_3X_FIELD} Use `id` instead."),
     )
-    voucher = graphene.Field(Voucher)
+    voucher = graphene.Field(Voucher, description="Voucher linked to the order.")
     voucher_code = graphene.String(
         required=False,
         description="Voucher code that was used for Order." + ADDED_IN_318,
     )
     gift_cards = NonNullList(
         GiftCard, description="List of user gift cards.", required=True
     )
-    customerNote = graphene.Boolean(required=True)
-    customer_note = graphene.String(required=True)
-    weight = graphene.Field(Weight, required=True)
-    redirect_url = graphene.String()
+    customerNote = graphene.Boolean(
+        required=True,
+        description="Indicate whether or not a customer has added a note to their order.",
+    )
```

Comment:
I wasn't aware of such a boolean field in the schema. I checked it now and it's actually covered by `customer_note = graphene.String` below. Also, nowhere in the backend we use it or set its value. I suggest removing this field at all on this occasion.

Reply:
removed 👍